src/receivers/AwsLambdaReceiver.ts

```diff
@@ -243,4 +247,10 @@ export default class AwsLambdaReceiver implements Receiver {
 
     return true;
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  private getHeaderValue(headers: Record<string, any>, key: string): string | undefined {
+    const caseInsensitivKey = Object.keys(headers).find((it) => key.toLowerCase() === it.toLowerCase());
```

Comment:
Lack of `e` here (re: `Insensitive`) intentional? 😄

Reply:
thanks for pointing this out to me :pray: and as it was not intentional I fixed it